### PR TITLE
mind the current number of lives when starting a new round. This fixes

### PR DESCRIPTION
**Pull request by rsBNT**
issue #1

### CODE REVIEW

File: spaceinvaders.py [hunk 1]
```diff
@@ -316,21 +316,32 @@ def reset(self, score, lives):
...
+	def reset_lives_sprites(self):
 		self.life1 = Life(715, 3)
 		self.life2 = Life(742, 3)
 		self.life3 = Life(769, 3)
-		self.livesGroup = sprite.Group(self.life1, self.life2, self.life3)
 		
+		if self.lives == 3:
+			self.livesGroup = sprite.Group(self.life1, self.life2, self.life3)
+		elif self.lives == 2:
+			self.lives -= 1
```
Comment (leerob) [inline spaceinvaders.py]: Is this supposed to be in here? Why -1 only on 2 lives?

Comment (rsBNT) [reply spaceinvaders.py]: Good spottin, must have been a c+p error. The lives are decremented in check_collisions() so no reason to decrement here. Decrementing worked fine before, just the sprites were not handled correctly on a new round

**Comment by leerob** [comment]
@rsBNT Merge conflicts

**Comment by rsBNT** [comment]
fixed, since i based my changes all on master, the order of merges might have introduced this

**Pull request merged**